### DISCUSSION TITLE
[ FF7] Crash Disc 2

### DISCUSSION THREAD

**Discussion by TheDrifter363-zz** [julianxhokaxhiu/FFNx#226] [General]
**Describe the bug**
Crashes when opening menu or entering chocobo ranch.

**To Reproduce**

1. Play on overworld for a few minutes.
2. Go to menu or enter town.
3. Freeze.

**Expected behavior**
Enter town normally or access menu normally.

**Screenshots**
N/A

**GPU (please complete the following information):**

- Brand Nvidia
- Driver Version 461.72

**Additional context**
Started when grinding for chocobos.

[crash_files.zip](https://github.com/julianxhokaxhiu/FFNx/files/6275587/crash_files.zip)

**Comment by julianxhokaxhiu** [comment]
Hi, can you please let me know how are you running the game? Using vanilla FFNx or 7th Heaven or any other solution integrating FFNx?

**Reply by TheDrifter363-zz** [reply]
Sure. I'm running vanilla ffnx, using satsuki's upscaled backgrounds. It comes with ffnx. I updated to your ffnx steam version, since I didn't wanna use his 98 exes. Only thing I didn't copy from your updated canary build, was the shaders folder. Satsuki told me not to copy them. Should I have? I copied everything else from 228.

**Reply by julianxhokaxhiu** [reply]
> Sure. I'm running vanilla ffnx, using satsuki's upscaled backgrounds. It comes with ffnx. I updated to your ffnx steam version, since I didn't wanna use his 98 exes. Only thing I didn't copy from your updated canary build, was the shaders folder. Satsuki told me not to copy them. Should I have? I copied everything else from 228.

I'm not sure I understood your setup. If you're using Satsuki's setup, please consider using everything he provides, as it provides it. Usually we work together so rest assured you will have always the latest canary that makes sense. If you want to update things manually or you change the setup as it was provided, you enter in a limbo which me neither Satsuki are able to help.

If you wish to use the Steam release as it is, consider doing a fresh setup and installing things manually. Otherwise you can consider using 7th or Satsuki's Steam launcher, which both basically runs FF7 using the 1998 edition EXEs, actually because there's literally no gain in running the Steam edition as it comes, unless you know what you're doing.

**Reply by TheDrifter363-zz** [reply]
> I'm not sure I understood your setup. If you're using Satsuki's setup, please consider using everything he provides, as it provides it. Usually we work together so rest assured you will have always the latest canary that makes sense. If you want to update things manually or you change the setup as it was provided, you enter in a limbo which me neither Satsuki are able to help.
> 
> If you wish to use the Steam release as it is, consider doing a fresh setup and installing things manually. Otherwise you can consider using 7th or Satsuki's Steam launcher, which both basically runs FF7 using the 1998 edition EXEs, actually because there's literally no gain in running the Steam edition as it comes, unless you know what you're doing.

Alright, just wanted check. I already mentioned on the forums, I didn't wanna use 7th heaven, since I'm not a big fan of its admin access and disc mounting setup. I run everything through steam, and admin access throws a wrench in my setup.

I'm using Satsuki's standalone, his 5.30. He actually provided instructions on what how to continue to use the steam executable with his latest version. I had to download your latest canary steam build, and overwrite everything except the shaders folder. I'll post a link.

http://forums.qhimm.com/index.php?topic=14938.10725
post 596

I also use new threat standalone. When it detects a 98 exe, it makes some patches, but it leaves the disc check in. I hate that, so that's why I'm having to use the steam exe. I don't want a disc check nor admin access.

I don't know if the files I provided are of any help, but there they are.

Using version 228 steam, overwrote everything except the shaders folder.

Edit: Adding another list of crash files.
[crash_files (2).zip](https://github.com/julianxhokaxhiu/FFNx/files/6293599/crash_files.2.zip)

**Reply by julianxhokaxhiu** [reply]
Please override everything, use the latest canary and if you're using NT approach the relative author. This has likely nothing to do with FFNx. The point where it crashes for you has been proven to work by a ton of other users.

As said, either use the solutions as they came out of the box, or if you're willing to do hybrid setups, then you're on your own, sorry.

**Reply by TheDrifter363-zz** [reply]
Hey Julian,

I think I figured it out man. Think the crashes might be related to new threat. I've had similar instability using new threat with the reunion. I let sega chief know. Usually when fast forwarding, and then it'll crash randomly. I didn't try your ffnx without his mod, but if I did, then I shouldn't crash, I think.

**Reply by julianxhokaxhiu** [reply]
Glad you figured it out. Please consider always testing FFNx in vanilla, and by vanilla I mean no other mods installed. If you can verify your issue even in that case, then it's worth to be thrown in here. Otherwise you may want to approach the mod author the crash comes from.
